howto/finetune_adapter.md

```diff
@@ -1,11 +1,13 @@
 # Finetuning with Adapter
 
-Adapter, first introduced for the LLaMA model as [LLaMA-Adapter](https://arxiv.org/abs/2303.16199), is a form of prefix-tuning that prepends a learnable adaption-prompt to the inputs of the attention blocks in an LLM. In total, there are only ~500M parameters to update during finetuning in LLaMA, which significantly reduces the memory footprint and speeds up training.
+Adapter, first introduced for the LLaMA model as [LLaMA-Adapter](https://arxiv.org/abs/2303.16199), is a form of prefix-tuning that prepends a learnable adaption-prompt to the inputs of the attention blocks in an LLM. In total, there are only ~500k parameters to update during finetuning in LLaMA, which significantly reduces the memory footprint and speeds up training.
```

Comment:
This updated number is for StableLM 3b, right? If so, we should update it

```suggestion
Adapter, first introduced for the LLaMA model as [LLaMA-Adapter](https://arxiv.org/abs/2303.16199), is a form of prefix-tuning that prepends a learnable adaption-prompt to the inputs of the attention blocks in an LLM. In total, there are only ~500k parameters to update during finetuning in StableLM 3B, which significantly reduces the memory footprint and speeds up training.
```

Reply:
Yes this is update number for StableLM 3B. Adapter V2 extends the number of parameters by an additional 1.5M